content/md/assets/about-product-link-rules.md

```diff
@@ -206,13 +206,15 @@ You can use multiple conditions to make your selection. Those conditions are cum
 Here is the list of the fields you can use to select your products:
 
 - Among the product properties:
-  * the `product family`,
-  * the `product categories`,
-  * the `product status`,
+  * the `family`,
+  * the `categories`,
+  * the `status`,
+  * the `enabled`.
 - Among the product attributes:
   * the `identifier` attribute,
-  * the `text` attributes,
-  * the `simple/multi select` attributes.
+  * the `text` & `text area` attributes,
+  * the `simple select` and `multi select` attributes,
+  * the `yes/no` attribute.
 
```

Comment:
We can do a selection on every attribute type so I don't know if we need to specify it

Reply:
Hmm this is weird, as the documentation is supposed to be exhaustive... ALL attributes are really usable in the selection?